ouroboros-consensus/ouroboros-consensus-mock/test/Test/Consensus/Ledger/Mock/Generators.hs

```diff
@@ -60,6 +64,10 @@ instance Arbitrary (HeaderHash blk) => Arbitrary (Point blk) where
   Generators
 -------------------------------------------------------------------------------}
 
+deriving via Positive Word64 instance Arbitrary SlotNo
+
+deriving via Word64          instance Arbitrary EpochNo
+
```

Comment:
For now, I'd create `Test.Util.Slotting.Orphans.Arbitrary` with these two instances.

Reply:
Yes, that's a much nicer approach